lib/kredis/types/counter.rb

```diff
@@ -1,20 +1,16 @@
 class Kredis::Types::Counter < Kredis::Types::Proxying
   proxying :multi, :set, :incrby, :decrby, :get, :del, :exists?
 
+  before_method :set_default, :value, :increment, :decrement
+
   attr_accessor :expires_in
 
   def increment(by: 1)
-    multi do |pipeline|
-      pipeline.set 0, ex: expires_in, nx: true
-      pipeline.incrby by
-    end[-1]
+    incrby by
```

Comment:
Isn't this a behavior change, then? Unless you manually specify the default to be zero?